### DISCUSSION TITLE
Multi-segment start point & ignore first pixel

### DISCUSSION THREAD

**Discussion by justinj57** [awawa-dev/HyperSPI#33] [General]
Is it required to start the LED run in the middle of the bottom row when using multi segment, or can I start in a corner? 

Also, if I use a sacrificial LED for each segment and choose the option to ignore the first pixel in HyperHDR,  will the first pixel be ignored in both segments, or just the first?

**Comment by awawa-dev** [comment]
Hi
You can start the LED anywhere you want and turn off (i.e. force always black color, sacrificial LEDs cannot be ignored!) any LED, as long as both segments are equal and you can set the LED geometry in HyperHDR. The wizard does not support artificially added sacrificial LEDs, but you can add them manually by editing the JSON in the appropriate places.
